src/ResourceManager/Intune/Commands.Intune/ApiVersionHandler.cs

```diff
@@ -0,0 +1,95 @@
+﻿// ----------------------------------------------------------------------------------
```

Comment:
@rnikhilkumar This should be a test artifact only, it should not be used by your cmdlets